src/components/Search/Search.stories.tsx

```diff
@@ -1,37 +1,51 @@
-import { Search } from '@components/Search'
-import { ComponentMeta, ComponentStory } from '@storybook/react'
+import type { Meta, StoryObj } from '@storybook/react'
 
-export default {
-  title: 'Search',
-  component: Search
-} as ComponentMeta<typeof Search>
-
-const Template: ComponentStory<typeof Search> = args => (
-  <div style={{ maxWidth: '400px' }}>
-    <Search {...args} />
-  </div>
-)
-
-export const Small = Template.bind({})
+import { Search } from './Search'
 
-Small.args = {
-  size: 'small'
+const meta: Meta<typeof Search> = {
```

Comment:
no tutaj też by się przydały kontrolki (jeżeli ich nie ma, ale możemy w tej chwili to odpuścić)

Reply:
poprawię tego Storisa, a Panel zrobię na nowo